core/config.lua

```diff
@@ -67,6 +67,12 @@ TC2.defaultConfig.bar = {
 	colorMod			= 0,									-- color modifier
 }
 
+-- self threat bar color settings
+TC2.defaultConfig.selfBar = {
```

Comment:
could you please change the naming to `playerBarCustomColor` here and everywhere else? selfBar doesn't make a lot of sense to me.

Reply:
Renamed